lib/new-admin/graphql/resolvers/log.resolver.js

```diff
@@ -1,18 +1,30 @@
 const { parseAsync } = require('json2csv')
+const moment = require('moment')
+const _ = require('lodash/fp')
 
 const logs = require('../../../logs')
 const serverLogs = require('../../services/server-logs')
 
+const dateFormat = (timezone, logs) => _.map(log => {
+  const offset = timezone.split(':')[1]
+  return {
+    ...log,
+    timestamp: moment.utc(log.timestamp).utcOffset(parseInt(offset)).format('YYYY-MM-DDTHH:mm:ss.SSS')
+  }
+}, logs)
+
```

Comment:
Do you think it's worth moving this dateFormat to a single place? it's pretty much the same as the function bellow and we might have to use this for future features. This would have to have a third argument with the list of fields to make it generic

Reply:
Hmm, that actually was something I was trying to do the first time around, but thought it might not be worth the trouble. See if you like my solution